datacenter/dtr/2.5/reference/cli/backup.md

```diff
@@ -34,6 +34,9 @@ Note:
   * This backup contains sensitive information and should be
     stored securely.
 
+  * Using the '--offline-backup' flag will temporarily shut down the rethinkdb container. 
+    You should take the replica out of your load balancer to avoid downtime.
+
```

Comment:
Do not use "should."

Reword:
"Remove the replica from your load balancer to avoid downtime.